src/test/ui/consts/const-eval/format_args.rs

```diff
@@ -0,0 +1,8 @@
+#![crate_type = "lib"]
+
+const A: std::fmt::Arguments = format_args!("literal");
```

Comment:
So this is insta-stable?

Reply:
i think it's just not gated in this WIP form of the PR

Reply:
> So this is insta-stable?

Yes. That's why I added `needs-fcp`.